modules/tensor_mechanics/tests/linear_elastic/tests

```diff
@@ -1,10 +1,4 @@
 [Tests]
-  [./tween]
-    type = 'Exodiff'
-    input = 'tween.i'
```

Comment:
I will remove it when `LinearElasticMaterial` is removed. I want to make the migration path as clear and simple as possible.